packages/transaction-manager/src/transactions-factory.ts

```diff
@@ -136,6 +139,62 @@ export default class TransactionsFactory {
 
     const encryptedDataSerialized: string = MultiFormat.serialize(encryptedData);
 
-    return { encryptedData: encryptedDataSerialized };
+    if (encryptionParams.length === 0) {
+      return { encryptedData: encryptedDataSerialized };
+    } else {
+      // TODO: Refactor, duplicate with createEncryptedTransactionInNewChannel
+      // Check that all the encryption parameters given are ECIES (the only encryption method supported for now)
+      if (
+        !encryptionParams.every(
+          (encryptionParam: EncryptionTypes.IEncryptionParameters) =>
+            encryptionParam.method === EncryptionTypes.METHOD.ECIES,
+        )
+      ) {
+        throw new Error(`encryptionParams method must be all: ${EncryptionTypes.METHOD.ECIES}`);
+      }
+
+      // TODO: Check if channel already includes new encryptionParams
```

Comment:
TODO here

Reply:
Removing this TODO.  We don't actually want to check if the channel already includes the new encryptionParams.  This would be an O(n) operation. This is not worth doing when adding new encrypted taransactions.

Reply:
Removed.